src/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsController.php

```diff
@@ -231,4 +232,35 @@ private function manageLegacyFlashes($messageId)
             );
         }
     }
+
+    /**
+     * Toggle hook status
```

Comment:
We need to secure this controller using `AdminSecurity` annotation, you can read about it here https://devdocs.prestashop.com/1.7/modules/concepts/controllers/admin-controllers/ 😉

Reply:
Here i think https://devdocs.prestashop.com/1.7/development/architecture/migration-guide/controller-routing/#security
Il will check ! Thanks

Reply:
Done !